lib/utilities/collectResponseHeaders.js

```diff
@@ -104,11 +104,13 @@ function collectResponseHeaders(objectMD, corsHeaders, versioningCfg,
         responseMetaHeaders['x-amz-replication-status'] =
             objectMD.replicationInfo.status;
     }
-    if (objectMD.replicationInfo &&
-        // Use storageType to determine if user metadata is needed.
-        objectMD.replicationInfo.storageType &&
-        Array.isArray(objectMD.replicationInfo.backends)) {
+    if (Array.isArray(objectMD?.replicationInfo?.backends)) {
         objectMD.replicationInfo.backends.forEach(backend => {
+            const locationConstraint = config.getLocationConstraint(backend.site);
+            if (!config.includeCrrStatusMetadata &&
```

Comment:
do we need this configurable?

- With CRR, it does not really make sense to report the "replicated version id", since it is the same: not explicit, but probably handled already by the conditions below?
- If we set it to false, we keep the current behavior: which is not acceptable. So we should check with s3c team (and PO) if reporting the status would be acceptable...